documentation/source/user_section/tutorials/lesion-analysis/lesion-statistics/before-starting.rst

```diff
@@ -0,0 +1,28 @@
+Before starting this tutorial
+#############################
+
+#. Make sure that you have the following files in your working directory:
```

Comment:
Before saying this, we should say that it can be done on your own data. Otherwise, the person will download the data and then realize that it didn't necessarily had to.

Reply:
good point! agree, updated in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4586/commits/45b5e46bf2b207e86410dbad0c45e871a65117f0